docs/package_info/contributing.rst

```diff
@@ -37,13 +46,15 @@ anyway so we can work with you to write the necessary tests.
 Running the Test Suite
 ~~~~~~~~~~~~~~~~~~~~~~
 
+
+
```

Comment:
Why the whitespace change?

Reply:
This shouldn't have been included. I was going to put the notice here originally.